bentoml/marshal/marshal.py

```diff
@@ -141,6 +141,7 @@ def __init__(
         self.setup_routes_from_pb(self.bento_service_metadata_pb)
         if psutil.POSIX:
             import resource
+
```

Comment:
update the docstring in MarshalService class to reflect the changes in this PR?

Reply:
The usage is almost the same as before.

Reply:
I thought we changed the semantic of `mb_max_latency`?

Reply:
previously it was closer to `max_wait_time`?

Reply:
@parano just updated the docstring